packages/utils/tailwind-plugins/utilities/index.js

```diff
@@ -12,5 +12,7 @@ module.exports = plugin.withOptions(options => ({ addUtilities, theme }) => {
         'transitionTimingFunction.in'
       )}, border-color ${theme('transitionDuration.300')} ${theme('transitionTimingFunction.in')}`,
     },
+    'no-scrollbar': { scrollbarWidth: 'none' },
+    'no-scrollbar::-webkit-scrollbar': { display: 'none' },
```

Comment:
```suggestion
    'u-no-scrollbar': {
      scrollbarWidth: 'none',
      'no-scrollbar::-webkit-scrollbar': {
        display: 'none'
      }
    },
```

Reply:
This also seems to work, wdyt?
```
'.no-scrollbar': {
      scrollbarWidth: 'none',
      '&::-webkit-scrollbar': { display: 'none' },
    },
```

Reply:
It would be better to follow the standard naming convention, `u-*`

Reply:
Wasn't aware of this naming convention, got it! :+1: